### ISSUE TITLE
Add banner to giveth.io homepage when we launch the GIVeconomy

### ISSUE THREAD

**Issue by laurenluz** [Giveth/giveth-next#519]
Here is the banner. Please note... the button should say "Check your GIVdrop" not "Check your airdrop"

![image.png](https://images.zenhubusercontent.com/5fdfac16a58523a866ddb68d/af7472b9-404a-4ea1-a891-f620649d2fc0)

This is what I want:
- [ ] Banner is live on the homepage when we launch
- [ ] when you click the pink button, you get sent to the GIVdrop claim page
- [ ] when you click anywhere else on the banner, you get sent to the GIVeconomy Overview page

Is this possible? @mateodaza 

## References:

The original design issue: https://github.com/Giveth/GIVeconomy/issues/158
The figma: @rodricastillo9013 can you put a link to this design in the figma please?

**Comment by laurenluz** [comment]
@markoprljic tagging you so you can be in the loop on this if you want

**Comment by rodricastillo9013** [comment]
Yes! https://www.figma.com/file/LKBHpnxI9sLjde7DzOQIYQ/Giveth-Marketing?node-id=18%3A12

**Comment by mateodaza** [comment]
Yes totally possible. Looks great :)

**Comment by divine-comedian** [comment]
small text fix - airdrop -> givdrop 
https://github.com/Giveth/giveth-next/commit/1053af221774d44d903dd2a51e55de859bbed535

![image](https://user-images.githubusercontent.com/67759413/146594826-17c7745e-3213-4d30-b992-aff42b9a9f27.png)

looks good to go! moving to DONE
